server/rest/src/main/java/org/hawkular/apm/server/rest/AnalyticsHandler.java

```diff
@@ -524,6 +560,42 @@ public void getTraceCompletionFaultCount(
 
     }
 
+    @POST
+    @Path("trace/completion/faultcount")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(
+            value = "Get the number of trace instances that returned a fault",
+            response = Long.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Success"),
+            @ApiResponse(code = 500, message = "Internal server error") })
+    public void getTraceCompletionFaultCount(
```

Comment:
There is a GET and a POST handler for `trace/completion/faultcount`. These two could reuse some code.